### DISCUSSION TITLE
Can we use it with react-bootstrap library..?

### DISCUSSION THREAD

**Discussion by Sandeep06Dev** [jaredpalmer/formik#2645] [Q&A] [answered]
Can we use it react-bootstrap..?

**Comment by maddhruv** [comment, marked as answer]
Hey @Sandeep0695 there is a fantastic blog/article by @Nicknyr on hackernoon that you can follow to use `react-bootstrap` with formik
https://hackernoon.com/building-react-forms-with-formik-yup-and-react-bootstrap-with-a-minimal-amount-of-pain-and-suffering-1sfk3xv8
